content/tutorials/guides/how-to-collect-product-variations.md

```diff
@@ -0,0 +1,277 @@
+</div>
+
+<div class="block-requirements">
+    <div class="block-requirements-headline">
+        If you're following our App workflow, make sure you previously read:
+    </div>
+    <div class="block-requirements-row">
+        <img src="../../img/illustrations/illus--Attributegroup.svg" width="110px">
+        <div class="block-requirements-steps">
+            <ul>
+                <li>Step 1. <a href="how-to-get-your-app-token.html" target="_blank" rel="noopener noreferrer">Get your App token tutorial</a></li>
+                <li>Step 2. <a href="how-to-retrieve-pim-structure.html" target="_blank" rel="noopener noreferrer">How to retrieve PIM structure</a></li>
+                <li>Step 3. <a href="how-to-get-families-and-attributes.html" target="_blank" rel="noopener noreferrer">How to get families and attributes</a></li>
+                <li>Step 4. <a href="how-to-get-product-information.html" target="_blank" rel="noopener noreferrer">How to get product information</a></li>
+            </ul>
+        </div>
+    </div>
+</div>
+
+## Context
+
+In the PIM we handle product models and product variations.
+
+![scheme_variants](../../img/tutorials/how-to-collect-product-variations/scheme_variants.png)
+
+
+Here are quick definitions:
+
+**A product model gathers products with many similarities but a few differences - these differences are called “variations” and are defined by an associated family variant.**
+
+*Example: T-shirts with the same material (”wool”), brand (”a brand”), and weight (”16 grams”); but with different sizes and colors.*
+
+**A product variant is associated with a product model. It contains the common attribute values of its product model plus values for variations.**
+
+*Example: A t-shirt with material = ”whool”, brand = ”a brand”, weight = ”16 grams”, size = “XL”, and color = “blue*”.
+
+And because a relation schema is worth a thousand words:
+
+![scheme_model_and_variant](../../img/tutorials/how-to-collect-product-variations/scheme_model_and_variant.svg)
+
+::: tips
+Get the big picture <a href="/getting-started/synchronize-pim-products-6x/step-0.html" target="_blank" rel="noopener noreferrer">here</a>.
+:::
+
+
+## Collect product variations
+
+### Workflow
+
+![schema_product_models](../../img/tutorials/how-to-collect-product-variations/schema_product_models.svg)
+
+In this tutorial, we will introduce you to the two use cases you may encounter for retrieving products with variations.
+
+- **Use case 1**: Collect all product variation information, just as they are in the PIM (up to 2 possible levels)
+- **Use case 2**: Collect all product variation information on 1 level only)
+
+::: info
+We noticed that many e-commerce solutions understand product variation on only one level. This means that for Akeneo a special recollection of the variations must be done to have them all on the same level.
+:::
+
+### 0 - Initialization
+
+```php [activate:PHP]
```

Comment:
This step in not exactly the same than in https://github.com/akeneo/php-guided-tutorials/pull/4/files#diff-4a45687bf67299eda240500899b81b7f57dae3b1014be4a7fe2f7acb7fbd8ac5R11 or https://github.com/akeneo/php-guided-tutorials/pull/4/files#diff-6637c0fc4b1cf3c5db58fbc4fda9e4fc0898392f8bb76e81597e14b08daea23cR29 . I let you harmonize it